src/classes/DoubleLinkedList.js

```diff
@@ -0,0 +1,77 @@
+class DoubleLinkedList {
+    constructor(){
+        this.first = null;
+        this.last = null;
+        this.len = 0;
+    }
+
+    push(node) {
+        this.len += 1;
+        if (this.first === null) {
+            this.first = node;
+            this.last = node;
+        } else {
+            this.last.next = node;
+            node.prev = this.last;
+            this.last = node;
+        }
+    }
+
+    unshift(node) {
+        this.len += 1;
+        if (this.first === null) {
+            this.first = node;
+            this.last = node;
+        } else {
+            this.first.prev = node;
+            node.next = this.first;
+            this.first = node;
+        }
+    }
```

Comment:
same here, what do you think?

```suggestion
    unshift(value) {
        const node = new Node(value);

        if (this.first === null) {
            this.first = node;
            this.last = node;
        } else {
            this.first.prev = node;
            node.next = this.first;
            this.first = node;
        }

        this.len += 1;
    }
```

Reply:
ok